android/settings.gradle

```diff
@@ -1,4 +1,4 @@
-rootProject.name = 'All About Olaf'
+rootProject.name = 'AllAboutOlaf'
```

Comment:
Does this change our project on Play Store or anything human-readable by end users?  Or is this just internal to the build and APK itself?

Reply:
AFAIK, it's internal to the build? Gradle yelled at me that spaces would no longer be supported in Gradle 5, so I can't imagine that it'd be user-facing.

Reply:
Ah, okay.